src/Sylius/Bundle/ApiBundle/CommandHandler/Checkout/AddressOrderHandler.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\CommandHandler\Checkout;
+
+use Doctrine\Persistence\ObjectManager;
+use SM\Factory\FactoryInterface as StateMachineFactoryInterface;
+use Sylius\Bundle\ApiBundle\Command\Checkout\AddressOrder;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\OrderCheckoutTransitions;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Webmozart\Assert\Assert;
+
+final class AddressOrderHandler
+{
+    /** @var OrderRepositoryInterface */
+    private $orderRepository;
+
+    /** @var FactoryInterface */
+    private $customerFactory;
+
+    /** @var ObjectManager */
+    private $manager;
+
+    /** @var StateMachineFactoryInterface */
+    private $stateMachineFactory;
+
+    public function __construct(
+        OrderRepositoryInterface $orderRepository,
+        FactoryInterface $customerFactory,
+        ObjectManager $manager,
+        StateMachineFactoryInterface $stateMachineFactory
+    ) {
+        $this->orderRepository = $orderRepository;
+        $this->customerFactory = $customerFactory;
+        $this->manager = $manager;
+        $this->stateMachineFactory = $stateMachineFactory;
+    }
+
+    public function __invoke(AddressOrder $addressOrder): OrderInterface
+    {
```

Comment:
`$addressOrder->orderTokenValue` is used in three places, it should be assigned to new variable